src/librustc_session/config.rs

```diff
@@ -1104,11 +1105,12 @@ pub fn parse_color(matches: &getopts::Matches) -> ColorConfig {
 ///
 /// The first value returned is how to render JSON diagnostics, and the second
 /// is whether or not artifact notifications are enabled.
-pub fn parse_json(matches: &getopts::Matches) -> (HumanReadableErrorType, bool) {
+pub fn parse_json(matches: &getopts::Matches) -> (HumanReadableErrorType, bool, bool) {
```

Comment:
Should `(HumanReadableErrorType, bool, bool)` become a real named type? Two `bool` is pretty ungainly.

Reply:
Good point.